tests/testsuite/cargo_remove/gc_patch/out/Cargo.toml

```diff
@@ -5,5 +5,8 @@ members = [ "my-member" ]
 name = "my-project"
 version = "0.1.0"
 
+[patch."[ROOTURL]/bar1"]
```

Comment:
btw like the last commit, I assume this is tied into the previous commit. Could you squash this in so each commit stands on its own with passing tests?

Reply:
will do